forge/test/operators/matmul/test_matmul_fuse_bias.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import numpy as np
+
+import torch
+from torch import nn
+
+import forge
+from forge.op.eval.common import compare_with_golden_pcc
+
+import os
+
+@pytest.mark.parametrize("dims", [
+    (1, 784), (4, 784), (16, 784), (32, 784), (64, 784)
+])
+def test_matmul_bias_fusion(dims):
```

Comment:
Can you also add a test where matmul and bias are fused? 

This will be good to check if your logic works. Also, what you can explore, try to use compile depth from config.py file :)) Using it you'll be able to reduce the compile depth of fused matmul + bias test, and get passing results even if TTIR doesn't supports it :))

P.S.
You can also have single test, but use parameters to toggle bias True/False :))